html-css/week1/Lubna/order-desktop/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="order.css">
+  <title>payment Form </title>
+</head>
+
+<body>
+  <header>
+    <h1>ready to pay</h1>
+    <p>Fill in your payment details</p>
+  </header>
+  <main class="container">
+    <section class="order-top">
+      <h2>Order Summary</h2>
+
+      <div class="total-row">
+        <div class="row1">
+          <p class="total">Total</p>
+          <p class="per">$333/month</p>
+        </div>
```

Comment:
Are those really 2 separate paragraphs, or could they be seen as one? As presentation this looks fine, but what would you say about this approach:
```suggestion
        <p class="row1">
          <span class="total">Total</span>
          <span class="per">$333/month</span>
        </p>
```